lib/scripting/stdlib/err.ts

```diff
@@ -67,11 +72,15 @@ export class Err extends VbsApi {
 	 * @param helpcontext The context ID identifying a topic within helpfile that provides help for the error. If omitted, the Visual Basic Help file context ID for the error corresponding to the Number property is used, if it exists
 	 * @see https://docs.microsoft.com/en-us/office/vba/language/reference/user-interface-help/raise-method
 	 */
-	public Raise(code: number, source: string = '', description: string = '', helpfile: string = '', helpcontext: string = ''): void {
+	public Raise(code: number, source?: string, description?: string, helpfile?: string, helpcontext?: string): void;
+	public Raise(codeOrError: number | VbsError, source: string = '', description: string = '', helpfile: string = '', helpcontext: string = ''): void {
 		if (this.doThrow) {
-			throw new Error(`Error ${code}: ${description}`);
+			if (typeof codeOrError === 'number') {
+				throw new Error(`Error ${codeOrError}: ${description}`);
```

Comment:
should we use VbsError here?

Reply:
No, this is the case where also the VBS would crash, so it doesn't matter how we crash. `VbsError` is when the script potentially handles it, so we need an error number the script might be checking against.